src/lib/mojaloop-requests/wso2auth.js

```diff
@@ -91,6 +92,10 @@ class WSO2Auth {
         }
         return this.token;
     }
+
+    stop() {
```

Comment:
Is this function just to support testing?

Reply:
currently yes, but probably need to refactor the code to be able to call this class as
`start -> getToken,getToken,... -> stop`